src/wizards/tokenSwapDealWizard/stages/tokenDetailsStage/tokenDetailsStage.ts

```diff
@@ -123,4 +125,11 @@ export class TokenDetailsStage {
     this.hasUnsavedChangesForPrimaryDetails = this.primaryDAOTokenDetails.filter(viewModel => viewModel.viewMode === "edit").length > 0;
     this.hasUnsavedChangesForPartnerDetails = this.partnerDAOTokenDetails.filter(viewModel => viewModel.viewMode === "edit").length > 0;
   }
+
+  private addDefaultValuesToRegistrationData(wizardType: WizardType) {
+    if (wizardType === WizardType.createPartneredDeal) {
```

Comment:
Wonder if `makeAnOffer` should also be included?
Eg. the PrimaryDaoStage treats them the same:

```ts
  getIsPartneredDeal(wizardType: WizardType) {
    switch (wizardType) {
      case WizardType.createPartneredDeal:
      case WizardType.makeAnOffer:
      case WizardType.editPartneredDeal:
        return true;
```
https://github.com/PrimeDAO/prime-deals-dapp/blob/c4fe40304a8ae68cabced9ab1d4976a2d966a1a4/src/wizards/tokenSwapDealWizard/stages/primaryDaoStage/primaryDaoStage.ts#L34

Reply:
Yup, seems so

![grafik](https://user-images.githubusercontent.com/30693990/157630081-792d2aa0-c75e-4dd3-adb7-d3736c5aa7de.png)